singer-connectors/tap-harvestr/requirements.txt

```diff
@@ -0,0 +1 @@
+git+https://github.com/alan-eu/pipelinewise-tap-harvestr.git@feat/harvestr-tap
```

Comment:
Currently pointing to a feature branch of harvestr tap.
:todo: change to main branch once the harvestr tap feature is merged